<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于神经网络的内涝分析系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>166</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>  模型</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="Button_SVM">
         <property name="text">
          <string>SVM分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="KNN">
         <property name="text">
          <string>临近算法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="random_forest">
         <property name="text">
          <string>随机森林</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logisticregression">
         <property name="text">
          <string>逻辑回归</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Decision_tree">
         <property name="text">
          <string>决策树</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flood_b">
         <property name="text">
          <string>神经网络 </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>831</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="import_text">
       <property name="text">
        <string>文件导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>170</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>数据分析</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="tongji">
         <property name="text">
          <string>统计分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cor">
         <property name="text">
          <string>相关性分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="corr_heat">
         <property name="text">
          <string>相关性热力图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>451</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>数据显示</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Accurancy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>SVM</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>KNN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>逻辑回归</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>330</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>神经网络</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>随机森林</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>280</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>决策树</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="processdata">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>数据预处理</string>
    </property>
   </widget>
  </widget>
  <action name="actiontongjixuefenxi">
   <property name="text">
    <string>统计分析</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>相关性分析</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>相关性热力图</string>
   </property>
  </action>
  <action name="actionSVM">
   <property name="text">
    <string>SVM</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>相关性分析</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>相关性热力图</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>临近算法</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>随机森林</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>神经网络 </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
